docs/contribute/new-snippets.md

```diff
@@ -24,6 +24,14 @@ git checkout -b code-snippets-for-framework
 
 Before adding your snippets, open the `docs` folder with your editor of choice. Get familiarized with the documentation, including how the snippets are organized and their contents.
 
+Then change your current directory path to the docs folder. Synchronize the documentation file changes with the Storybook website by running the script `sync.js`.
+
+```shell
+cd docs && node sync.js
```

Comment:
Probably worth pointing out that this is an ongoing process, so they should run it in a separate terminal window.

Reply:
May be by converting the `sync.js` into a task will simplify more the docs.